lib/internal/bootstrap/loaders.js

```diff
@@ -120,7 +120,6 @@
     this.exportKeys = undefined;
     this.loaded = false;
     this.loading = false;
-    this.script = null;  // The ContextifyScript of the module
```

Comment:
This breaks the code cache generation (although it is already broken by https://github.com/nodejs/node/pull/23025). An easy way to test it is to run `./node --expose-internals tools/generate_code_cache.js  /tmp/node_code_cache.cc`. To fix it we need to have another way to expose the wrapped source again somehow and compile their cache in `internal/bootstrap/cache.js`.

(I'll fix the script and add a test)

Reply:
should we perhaps add a `// TODO(@joyeecheung): figure out way to expose wrapped source for code cache`; or do you see this as a blocker for @ryzokuken's work.

Reply:
For the record, I'm okay either way, I could look into it in another PR, leave it for @joyeecheung to do or add it in this one.

I'd not prefer to block this for it though since it's also kinda broken?

Reply:
I didn’t mean to block this so this is just a comment :) I am good with leaving a TODO here